src/app/(platform)/events/page.tsx

```diff
@@ -1,10 +1,11 @@
+import { EventsList } from '@/components/events/events-list';
 import { Heading2 } from '@/components/ui/heading-2';
 
-const Events = () => (
+const EventsPage = () => (
   <div className="mt-4 md:px-20">
     <Heading2>Eventos</Heading2>
-    <div className="mt-4">Próximamente.</div>
+    <EventsList></EventsList>
```

Comment:
Podemos simplificar este código utilizando una sola etiqueta
```suggestion
    <EventsList />
```